docs_sources/erp_fhirversion_change_20250115-source.adoc

```diff
@@ -0,0 +1,117 @@
+= E-Rezept FHIR Veränderungen für 20250115 image:gematik_logo.png[width=150, float="right"]
+include::./config-source.adoc[]
+
+Ab 15.01.2025 ist das gematik Workflow Package 1.4.0 gültig. Da sowohl DiGA als auch eML Teil des Packages sind, müssen einige Veränderungen vorgenommen werden. Diese Seite bietet eine Zusammenfassung der Veränderungen für die verschiedenen Hersteller.
```

Comment:
Das fühlt sich an als ob noch ein paar mehr Infos hier rein könnten und auch der Einstieg etwas leichter. Mehr Einführung in das Thema geben und wofür die Seite da ist

Reply:
Passt das?